commcare_connect/opportunity/views.py

```diff
@@ -291,6 +291,11 @@ def get_queryset(self):
         opportunity = get_opportunity_or_404(org_slug=self.request.org.slug, pk=opportunity_id)
         return OpportunityAccess.objects.filter(opportunity=opportunity).order_by("user__name")
 
+    def get_table_kwargs(self):
```

Comment:
Why not implement this on a class that subclasses `SingleTableView` and use that everywhere to be DRY?

Reply:
Thanks @sravfeyn added the changes.